rationale.md

```diff
@@ -18,16 +28,24 @@ For example, libraries that are instrumented with `opentelemetry-api 1.0.1` will
 
 ### SDK Stability:
 
-Public portions of the SDK (constructors, configuration, end-user interfaces) must remain backwards compatible.
-Internal interfaces are allowed to break.
+Public portions of the SDK (constructors, configuration, end-user interfaces) must remain backwards compatible. Internal interfaces are allowed to break.
+
+## Core components
+
+Core components refer to the set of components which are required as per the spec. This includes API, SDK, and exporters which are required by the
+specification. These exporters are OTLP, Jaeger, Zipkin, Console and InMemory.
```

Comment:
Are the console and inmemory exporters defined in the spec?

Reply:
Removed